src/quo2/components/reactions/reaction.cljs

```diff
@@ -60,11 +58,11 @@
                                            :border-width 1)))}
      [icons/icon emoji {:no-color true
                         :size     16}]
-     [quo2.text/text {:size :paragraph-2
-                      :weight :semi-bold
-                      :color text-color
-                      :flex-direction :row
-                      :align-items :center
+     [quo2.text/text {:size            :paragraph-2
```

Comment:
Just aliasing as `text` instead of `quo2.text` seems better because the `reactions` component doesn't need to know there are two versions of the comp. library.

Reply:
yeah i just moved this component, just didn't review it